src/wms/src/test/java/org/geoserver/wms/wms_1_1_1/DimensionsVectorCapabilitiesTest.java

```diff
@@ -419,4 +473,43 @@ public void testCustomContinuousDate() throws Exception {
         assertXpathEvaluatesTo(
                 "2011-05-01T00:00:00Z", "//Layer[Name='sf:TimeElevation']/Extent/@default", dom);
     }
+
+    @Test
+    public void testElevationContinuousChallenge() throws Exception {
+        Catalog catalog = getCatalog();
+        TestResourceAccessManager tam = getResourceAccessManager();
+
+        FeatureTypeInfo featureTypeInfo = catalog.getFeatureTypeByName("sf:TimeElevation");
+        tam.putLimits(
+                "admin2",
+                featureTypeInfo,
+                new VectorAccessLimits(
+                        CatalogMode.CHALLENGE, null, Filter.EXCLUDE, null, Filter.EXCLUDE));
+
+        setupVectorDimension(
```

Comment:
Shouldn't this security setup be reverted at the end of the test, or in a @before method?

Reply:
Indeed, makes sense, adding on an @After method.